packages/cli/package.json

```diff
@@ -168,13 +170,17 @@
   "dependencies": {
     "@prisma/config": "workspace:*",
     "@prisma/engines": "workspace:*",
-    "@prisma/studio-core-licensed": "0.6.0",
+    "@prisma/studio-core-licensed": "0.0.0-dev.202511120235",
```

Comment:
stalling on releasing v0.7 until SQLite adapter is finished - currently only does introspection.

follow up PRs will bump the version.